js/src/dashboard/index.scss

```
@@ -45,6 +45,12 @@
 			display: flex;
 			flex-direction: column;
 			flex: 1;
+
+			> div:first-of-type {
+				display: flex;
+				flex-direction: column;
+				flex: 1;
+			}
```

A part of the style looks redundant. Maybe this would be enough?
```suggestion
			flex: 1;

			> div:first-of-type {
				display: flex;
				flex-direction: column;
			}
```